tracing-opentelemetry/src/subscriber.rs

```diff
@@ -327,12 +331,22 @@ where
     {
         OpenTelemetrySubscriber {
             tracer,
+            event_location: self.event_location,
             tracked_inactivity: self.tracked_inactivity,
             get_context: WithContext(OpenTelemetrySubscriber::<C, Tracer>::get_context),
             _registry: self._registry,
         }
     }
 
+    /// Sets whether or not event span's metadata should include detailed location
+    /// information, such as the file, module and line number.
```

Comment:
Minor nitpick: we should probably document that the default is `true` here?
```suggestion
    /// information, such as the file, module and line number.
    ///
    /// By default, event locations are enabled.
```